airgun/views/oscapreports.py

```diff
@@ -0,0 +1,47 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Table
+from widgetastic.widget import Text
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import ActionsDropdown
+
+
+class OSCAPReportsView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Compliance Reports']")
+    delete_reports = Text("//div[@id='submit_multiple']")
+    select_all = Checkbox(locator="//input[@id='check_all']")
+    table = Table(
+        './/table',
+        column_widgets={
+            0: Checkbox(
+                locator=".//input[@class='host_select_boxes']"),
+            'Host': Text('./a'),
+            'Reported At': Text('./a'),
+            'Policy': Text('./a'),
+            'Openscap Capsule': Text('./a'),
+            'Passed': Text('./a'),
+            'Failed': Text('./a'),
+            'Other': Text('./a'),
```

Comment:
FYI, you don't need to model columns that you don't plan to interact with.
AFAIR widgetastic can still read all columns, also by name, and will assume `Text()` when column is not declared explicitly. So you can still read values in your test.

Reply:
ahh thanks for explaining, just kept those columns which I think I would need to interact with.